query/checkpointing.go

```diff
@@ -67,6 +67,21 @@ func (c *QueryClient) BlsPublicKeyList(epochNumber uint64, pagination *sdkqueryt
 	return resp, err
 }
 
+// RawCheckpoints queries the checkpointing module for a set of raw checkpoints
+func (c *QueryClient) RawCheckpoints(pagination *sdkquerytypes.PageRequest) (*checkpointingtypes.QueryRawCheckpointsResponse, error) {
```

Comment:
If we can query each module using its query client, do we need these methods?

Reply:
We add them here for UX as wrappers. But not required. We can decide to remove all of them if we want, but I suppose that's a decision for another PR.